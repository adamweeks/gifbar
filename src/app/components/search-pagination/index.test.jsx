import React from 'react';
import renderer from 'react-test-renderer';

import SearchPagination from '.';

describe(`SearchPagination component`, () => {
  const amount = 10;
  const changeOffset = jest.fn();
  const currentOffset = 30;
  const forwardAvailable = true;
  const previousAvailable = true;
  const totalResults = 200;
  const component = renderer.create(
    <SearchPagination
      amount={amount}
      changeOffset={changeOffset}
      currentOffset={currentOffset}
      forwardAvailable={forwardAvailable}
      previousAvailable={previousAvailable}
      totalResults={totalResults}
    />
  );

  it(`renders properly`, () => {
    expect(component).toMatchSnapshot();
  });
});
