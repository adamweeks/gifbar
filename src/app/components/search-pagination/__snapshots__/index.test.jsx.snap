// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchPagination component renders properly 1`] = `
<div
  id="pagination"
>
  <div
    className="container"
  >
    <div
      className="item"
    >
      <button
        disabled={false}
        onClick={[Function]}
      >
        Prev
      </button>
    </div>
    <div
      className="item"
    >
      <button
        disabled={false}
        onClick={[Function]}
      >
        Next
      </button>
    </div>
    <div
      className="item"
    >
      Results: 
      200
    </div>
  </div>
</div>
`;
